prediction/src/tests/test_endpoints.py

```diff
@@ -4,14 +4,21 @@
 
     Provides unit tests for the API endpoints.
 """
-from functools import partial
 import json
+import os
+from functools import partial
 
 import pytest
-
 from flask import url_for
-from src.factory import create_app
 from src.algorithms import classify, identify, segment
+from src.factory import create_app
+
+
+def skip_slow_test():
+    """
+    Skip the wrapped test function unless the environment variable RUN_SLOW_TESTS is set.
+    """
+    return os.environ.get('RUN_SLOW_TESTS') != '1'
```

Comment:
In cases like this, I think `not os.environ.get('RUN_SLOW_TESTS')` is more flexible as it doesn't require the value to be strickly `'1'`. Also, `'0'` is a boolean `True`, so it should probably be set to an int ;).

Reply:
Just to clarify... any the following could be valid `RUN_SLOW_TESTS` values

`True`
`1`
`'any non empty string'`